solutions/devicesimulation-nohub/single-vm/setup.sh

```diff
@@ -185,6 +186,7 @@ echo ""
 echo "# Format: true | false"                                                                            >> ${ENVVARS}
 echo "# empty => Auth required"                                                                          >> ${ENVVARS}
 echo "export PCS_AUTH_REQUIRED=\"\""                                                                     >> ${ENVVARS}
+echo "export PCS_AUTH_WEBSERVICE_URL=\"\""                                                               >> ${ENVVARS}
```

Comment:
What's the behavior when this value is empty? There is no request sending to config service?

Reply:
Auth service is not used by DS. There won't be any requests made.